ui/components/MesheryPerformance/PerformanceProfiles.js

```diff
@@ -198,7 +213,7 @@ function PerformanceProfile({ updateProgress, enqueueSnackbar, closeSnackbar })
               </Button>
             </div>
           )}
-          <div style={{ justifySelf : "flex-end", marginLeft : "auto", paddingLeft : "1rem" }}>
+          <div style={{ justifySelf : "flex-end", marginLeft : "auto", paddingLeft : "1rem" }} onClick={() => setSearch("") }>
```

Comment:
oh styles here and there, should we use makeStyles?

Reply:
We can, but it might be difficult, as we do not have any definitive class name for these divs, that have style in it.

Reply:
Let's start now. Drive clarity. Give names to the classes. Propagate their use.